cfgov/jinja2/v1/_includes/organisms/image-text-25-75-group.html

```diff
@@ -1,3 +1,26 @@
+
+{# ==========================================================================
+
+   Image and Text 25/75 Group
+
+   ==========================================================================
+
+   Description:
+
+   See [GHE]/CFPB/hubcap/wiki/25-75-Info-unit-group
+
+   Create an info-unit-group organism when given:
+
+   value: Object defined from a StreamField block.
+
+   value.heading: A string containing title of module.
+
+   value.image_texts: TODO: add description for this.
+
+   value.link_image_and_heading: TODO: add description for this.
+
+   ========================================================================== #}
+
```

Comment:
Thanks for going to the trouble of adding this full comment, but this is deprecated and due to be removed at some point. Perhaps the comment should say that and point people toward `info-unit-group-2.html`?

Reply:
Added note in code comment.